ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/embeddedActivityBlock/embeddedActivityBlock.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { EmbeddedActivityBlockComponent } from 'ddp-sdk';
```

Comment:
It is in same project, so this import probably will result in circular reference error (have not checked yet). Try relative path instead.

Reply:
But if this is just the generated test and we are not writing a real one, then just delete. A test would be nice, though!